CONTRIBUTING.md

```diff
@@ -50,6 +50,34 @@ Any changes to the code base should follow the style and coding conventions
 used in the rest of the project. The version history should be clean, and
 commit messages should be descriptive and [properly formatted][commit-messages].
 
+Testing
+-------
+
+### Accessibility
+Testing for accessibility is a great way to make sure that all hackathon 
+enthusiasts can use Quill, regardless of ability. It's good practice to run 
+accessibility tests on any changes that you've made to ensure that no new 
+accessibility errors were introduced.
+
+An accessibility testing tool, pa11y-ci, has been provided and configured for 
+this project. To run pa11y-ci, make sure that Quill is running locally on 
+http://localhost:3000/ (alternatively, you can change the URLs specified in 
+`.pa11yci` to match those of your running instance). Then, run the command 
+`npm run test:accessibility`. If several of the URLs checked by pa11y-ci 
+produce the same number of errors, pa11y-ci may be having trouble logging in 
+with the default admin credentials specified in `.env`. Check that your 
+instance of Quill is running correctly, or change the credentials used in 
+`.pa11yci`.
+
+If your contribution adds any new pages to Quill, please add them to 
+`.pa11yci` to make sure that these pages are covered by the accessibility 
+tests. If your new pages are accessed as a non-logged-in user, add them at the 
+beginning of the URL list. If they are accessed when logged in, add them after 
+the URL with actions to log in.
+
+For more information on pa11y-ci, please visit [pa11y-ci] and [pa11y], 
+in particular the [section on actions][pa11y-actions].
+
```

Comment:
I think these comments should probably be in the README.md file, not in contributing. @staceybeard

Reply:
Sure, I'll move them.

Reply:
Done